transaction.go

```diff
@@ -24,6 +24,8 @@ type Transaction struct {
 	Customer                    *Customer                 `xml:"customer,omitempty"`
 	BillingAddress              *Address                  `xml:"billing,omitempty"`
 	ShippingAddress             *Address                  `xml:"shipping,omitempty"`
+	TaxAmount                   *Decimal                  `xml:"tax-amount,omitempty"`
+	TaxExempt                   bool                      `xml:"tax-exempt,omitempty"`
```

Comment:
Ah yeah, I see that now too, good catch. Since it always seems to be received, it'll be easier to use as a simple `bool`.